README.md

```diff
@@ -1,2 +1,64 @@
-# scope-plugin-generator
-This is a plugin generator for [Weave Scope](https://github.com/weaveworks/scope)
+# Scope Plugin Generator
+The Scope Plugin Generator uses [Cookiecutter](https://github.com/audreyr/cookiecutter) to automatically generate a [Weave Scope](https://github.com/weaveworks/scope) plugin's basic structure.
+
+You can start developing your own Scope plugins in less then 5 minutes by just following the instructions in the next section.
+
+## Plugin Bootstrapping
+To generate your own plugin you will need Cookiecutter, you can install it following these [instructions](https://cookiecutter.readthedocs.io/en/latest/installation.html).
+
+After you have installed Cookiecutter, you can create a basic Scope plugin structure using the following command: `cookiecutter https://github.com/weaveworks-plugins/scope-plugin-generator`.
+You will be prompted to insert the following basic information about your plugin:
```

Comment:
I can do multiple incremental templates one that generate the code, one for the dockerfile and makefile.